models/staking/ethereum/staking_ethereum_schema.yml

```diff
@@ -15,13 +17,32 @@ models:
             - depositor_address
             - tx_from
             - pubkey
+            - withdrawal_credentials
     columns:
       - name: depositor_address
-        description: "Address of indentified depositor entity"
+        description: "Address of identified depositor entity"
+        tests:
+          - unique:
+              config:
+                where: "depositor_address is not null"
```

Comment:
thanks again @0xRobin for help here -- this will be another good example to include in a "best practices guide" to spellbook